### PR TITLE
[UT] Introduce SSB Benchmark for Materialized View 

### PR DESCRIPTION
**Pull request by LiShuMing**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [x] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：
- Introduce SSB Benchmark for Materialized View 
   -  Query4-x now is supported to rewrite by flat mv.
   - Query1-x ~ Query3-x is ignored because view-delta is needed, to be supported later.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

File: fe/fe-core/src/test/resources/sql/materialized-view/test_mv.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+CREATE TABLE depts(
```
Comment (mofeiatwork) [inline fe/fe-core/src/test/resources/sql/materialized-view/test_mv.sql:1]: duplicated with `fe/fe-core/src/test/resources/sql/materialized-view/test_mv.sql`

File: fe/fe-core/src/test/resources/sql/materialized-view/test_mv.sql [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                          PRIMARY KEY (locationid)
+);
+CREATE TABLE emps(
+                     empid INT NOT NULL,
+                     deptno INT NOT NULL,
+                     locationid INT NOT NULL,
+                     empname VARCHAR(20) NOT NULL,
+                     salary DECIMAL (18, 2),
+                     PRIMARY KEY (empid),
+                     FOREIGN KEY (deptno) REFERENCES depts(deptno),
```
Comment (mofeiatwork) [inline fe/fe-core/src/test/resources/sql/materialized-view/test_mv.sql:18]: `FOREIGN KEY` ?

**Review by murphyatwork — DISMISSED** [review]

**Comment by LiShuMing** [comment]
run starrocks_fe_unittest

**Comment by LiShuMing** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 6 / 6 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/SessionVariable.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/rule/transformation/materialization/MaterializedViewRewriter.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/optimizer/rule/transformation/materialization/MvUtils.java|2|2|100.00%|[]|

**Review by ABingHuang — APPROVED** [review]
LGTM

**Review by kangkaisen — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by murphyatwork — APPROVED** [review]

**Comment by LiShuMing** [comment]
run starrocks_admit_test

**Pull request merged**